megatron/model/mamba/mamba.py

```diff
@@ -13,10 +13,8 @@
     from causal_conv1d import causal_conv1d_fn
     import einops
 except ModuleNotFoundError:
-    print(
-        "Unable to import Mamba kernels. Install them from our requirements/requirements-mamba.txt, or directly from https://github.com/state-spaces/mamba"
-    )
-    pass
+    assert False, "Unable to import Mamba kernels. Install them from our requirements/requirements-mamba.txt, \
```

Comment:
This will assert for all users, not just those wanting to run Mamba models. We should move this to only trip the assert if a Mamba layer is initialized and we still get a ModuleNotFoundError

Reply:
I just removed this change for now, we can add something later but I think the other changes are more important